src/main/java/group1/mips_simulator/components/cpuParts/CPU.java

```diff
@@ -43,6 +43,10 @@ public CPU(){
         romLoader = new RomLoader();
     }
 
+//    public void run() {
+//        this.romLoader.loadROM();
+//    }
```

Comment:
Why was this commented out? Should it be removed?

Reply:
yea, ill remove it
i dont remember what i initially put that in here for, but im offloaded a lot of ROM related capability to the RomLoader class itself